version-migration/v0.7-to-v0.8-migration/examples/src/test/kotlin/examples/BreakingChanges.kt

```diff
@@ -0,0 +1,120 @@
+package examples
+
+import com.amazon.ion.IonSystem
+import com.amazon.ion.system.IonSystemBuilder
+import com.amazon.ionelement.api.emptyMetaContainer
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueFactory
+import org.partiql.lang.eval.ExprValueType
+import org.partiql.lang.eval.TypedOpBehavior
+import org.partiql.lang.planner.PassResult
+import org.partiql.lang.syntax.Lexer
+import org.partiql.lang.syntax.Parser
+import org.partiql.lang.syntax.SqlLexer
+import org.partiql.lang.syntax.SqlParser
+import org.partiql.lang.types.CustomType
+import kotlin.test.Test
+
+class BreakingChanges {
+
+    /**
+     * Replaced: [PartiqlAst.Expr.Except], [PartiqlAst.Expr.Intersect], and [PartiqlAst.Expr.Union]
+     */
+    @Test
+    fun `Bag Operators (Replaced)`() {
+        fun `Bag Operators Listed`(bagOp: PartiqlAst.Expr): Nothing = when (bagOp) {
+            is PartiqlAst.Expr.Except -> TODO("TO BE REPLACED")
+            is PartiqlAst.Expr.Intersect -> TODO("TO BE REPLACED")
+            is PartiqlAst.Expr.Union -> TODO("TO BE REPLACED")
+            else -> TODO("")
+        }
+
+        fun `Example Usage`(): PartiqlAst.Expr.Intersect = PartiqlAst.build {
+            val setQuantifier: PartiqlAst.SetQuantifier = all()
+            val leftBag: PartiqlAst.Expr = bag()
+            val rightBag: PartiqlAst.Expr = bag()
+            intersect(
+                setq = setQuantifier,
+                operands = listOf(leftBag, rightBag),
+                metas = emptyMetaContainer()
+            )
+        }
+    }
+
+    /**
+     * Added: [PassResult]
```

Comment:
I know it's mentioned in the `@Test` function, but would be good to clarify that `PassResult` has been renamed in v0.8.0 and is part of the experimental planner API